### DISCUSSION TITLE
Docker Installation mit Macvlan in Verbindung mit MQTT und NodeRed

### DISCUSSION THREAD

**Discussion by MatzeG** [camueller/SmartApplianceEnabler#591] [Installation und Betrieb]
Hallo 
ich hätte eine Frage, wie ich das Zusammenspiel vom Smart Appliance Enabler mit meinem bestehenden MQTT und nodeRed Container aufziehe. 
Wenn ich es richtig verstanden haben, so brauche ich wg. der MultiCast Nachrichten vom SHM das macvlan-Netz. Derzeit habe ich für nodered das im SHM auf direkte Zähler Kommunikation: umkonfiguriert. Testweise konnte ich den SAE per Docker Compose starten möchte aber eigentlich keine 2 MQTT Instanzen betreiben. Damit ich im nodered auch auf die EnergyMeter Daten mitlesen kann tendiere ich derzeit dazu sowohl den MQTT wie auch die bestehende nodered Installation in das macvlan-Netz zu schieben. Gute Idee?
Hat jemand diese Konstellation oder kann mir einen Tip geben wie ich das umsetzen kann/sollte?

Gruß aus OWL

**Comment by camueller** [comment]
Kannst Du machen. Du musst ja nur nodered zur Docker-Compose-Datei hinzufügen.
